src/providers/auth.ts

```diff
@@ -100,14 +100,14 @@ export class UserBuilder {
     private userOptions?: UserOptions
   ) {}
 
-  /** Respond to the creation of a Firebase Auth user. */
+  /** @public Respond to the creation of a Firebase Auth user. */
```

Comment:
"Responds" since we pretend an implicit "this method..."

Reply:
Done